dev-lang/mujs/mujs-0_p20161031.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils flag-o-matic toolchain-funcs vcs-snapshot
```

Comment:
By checking whether any functions from `eutils` are used. For example `epatch` is from `eutils`. For this ebuild, trial and error should be enough to see whether `eutils` is needed.